pkgs/applications/science/electronics/librepcb/default.nix

```diff
@@ -32,9 +35,18 @@ stdenv.mkDerivation rec {
     wrapQtAppsHook
     opencascade-occt
     libGLU
+    rustPlatform.cargoSetupHook
+    cargo
+    rustc
   ];
   buildInputs = [ qtbase ];
 
+  cargoDeps = rustPlatform.importCargoLock {
+    lockFile = ./Cargo.lock;
```

Comment:
upstream has a lockfile, why do we not use it?

Reply:
Sure, done.